README.md

```diff
@@ -29,8 +29,9 @@ ChugSplash makes it easy to deploy and upgrade smart contracts **securely**. It'
 
 ## Documentation
 
-1. [ChugSplash File](https://github.com/chugsplash/chugsplash/blob/develop/docs/chugsplash-file.md): Detailed explanation of the file where you define a deployment or upgrade. Start here.
-2. [Defining Variables in a ChugSplash File](https://github.com/chugsplash/chugsplash/blob/develop/docs/variables.md): Comprehensive reference that explains how to assign values to every variable type in a ChugSplash file.
+1. [Setting up a ChugSplash project](https://github.com/chugsplash/chugsplash/blob/develop/docs/setup-project.md): Take your first steps with ChugSplash.
```

Comment:
Would be good to also update the Tutorial section to link to this guide as well.

Reply:
Was actually going to delete the tutorial since this serves the same purpose with a better UX. There are only two things included in the tutorial that aren't covered in this guide: immutables and a technical overview. Was planning on adding docs for both of these

Reply:
That sounds good to me